os/include/tinyara/fs/smart_procfs.h

```diff
@@ -68,7 +68,8 @@
 
 #define SMART_DEBUG_CMD_SET_DEBUG_LEVEL   1
 #define SMART_DEBUG_CMD_SHOW_LOGMAP       2
-
+#define SMART_DEBUG_CMD_DUMP_SECTOR       3
```

Comment:
What is the use of `SMART_DEBUG_CMD_DUMP_SECTOR` ?

Reply:
It is arg value of ioctl of smart.c, please see line4845 of smart.c

Reply:
Found it, thankyou